src/AssetManager/View/Helper/Asset.php

```diff
@@ -92,16 +109,24 @@ public function __invoke($filename)
             $cacheConfig = $this->config['caching']['default'];
         }
 
-        // if nothing done, return the original filename
-        if (!isset($cacheConfig['options']['dir'])) {
-            return $filename;
-        }
-
         // query string params
         $queryString = isset($this->config['view_helper']['query_string'])
             ? $this->config['view_helper']['query_string']
             : '_';
```

Comment:
It has been a while since I "Zend frameworked" (it's a verb now), but I think you might nor need the default underscore here. Configs get merged and overwritten left to right, so this value _should_ be set in the module config. Also, I know I'm a PR too late with mentioning this haha